### DISCUSSION TITLE
Is there a way to display the output of `println` statements from a function in a worksheet?

### DISCUSSION THREAD

**Discussion by isti115** [scalameta/nvim-metals#680] [Q&A] [answered]
For example, in this video you can see how adding a `println` statement results in the output appearing on hover of the executed function: https://youtu.be/Jv9B1crzpWM?t=16m20s
I can't seem to be able to access this information anywhere, `nvim.lsp.buf.hover()` only gives me the inferred type information. Am I missing something, or is this functionality not yet available through this plugin?

**Comment by ckipp01** [comment, marked as answer]
Hey @isti115 you'll want to map this to something https://github.com/scalameta/nvim-metals/blob/1b87e6bfa4174b5fbaee9ca7ec79d8eae8df7f18/doc/metals.txt#L907 and then you'll be able to see the worksheet output in a separate hover.

**Reply by isti115** [reply]
Oh, thank you, I don't know how I missed this... :man_facepalming: Sorry for the incomplete search on my part and thanks for the quick answer as well as the awesome plugin! :clap:
